### DISCUSSION TITLE
Can we update multiple collections in one query alike updateAll where some prop

### DISCUSSION THREAD

**Discussion by MuhammadKhizar7** [pocketbase/pocketbase#3736] [Q&A] [answered]
Can we update multiple collections in one query, like updateAll where some prop? For example, in the messages table, where there is a prop is_read and chat_id, using chat_id to update all unread messages to is_read = true

**Comment by ganigeorgiev** [comment, marked as answer]
No, there is no such builtin functionality at the moment.

If you need to execute bulk operations you can [register a new route](https://pocketbase.io/docs/js-routing/#registering-new-routes) and [perform a custom db operation](https://pocketbase.io/docs/js-database/).
